hopsworks-rest-utils/src/main/java/io/hops/hopsworks/restutils/RESTCodes.java

```diff
@@ -1640,7 +1640,11 @@ public enum FeaturestoreErrorCode implements RESTErrorCode {
     FEATURE_NAME_NOT_FOUND(210, "The Feature Name was not found in this version of the Feature Group.",
         Response.Status.BAD_REQUEST),
     VALIDATION_RESULT_IS_NOT_VALID_JSON(211, "The validation result is not a valid json.", 
-      Response.Status.BAD_REQUEST);
+      Response.Status.BAD_REQUEST),
+    FEATURE_OFFLINE_TYPE_NOT_PROVIDED(212, "Feature offline type cannot be null or empty.",
```

Comment:
This line should be removed?

Reply:
wait, its there, it was just `;` that was replaced with `,`

Reply:
It's not in the master. https://github.com/logicalclocks/hopsworks/blob/master/hopsworks-rest-utils/src/main/java/io/hops/hopsworks/restutils/RESTCodes.java#L1642